### PR TITLE
update docs

### PR DESCRIPTION
**Pull request by kfu02**
## Description
A refresh of the docs for Spring 2023. New FAQ page based on commonly asked questions I've heard (feel free to add more).

https://rj-rc-software.readthedocs.io/en/doc-update/

## Steps to Test
Read newly added docs; specifically, FAQ and Troubleshooting.

**Expected result:** docs are clear.

## Key Files to Review
All, but ignore all files in `old_docs/`. (Would recommend looking at the RTD generated website first.)

## Review Checklist

- [x] **Docstrings**: All methods and classes should have the file appropriate docstrings which follow the guidelines in the ["Contributing" page](https://rj-rc-software.readthedocs.io/en/latest/contributing.html) of our docs.
- [x] **Remove extra print statements**: Any print statements used for debugging should be removed
- [x] **Tag reviewers**: Tag some people for review and ping them on Slack

### CODE REVIEW

File: docs/source/faq.rst [hunk 1]
```diff
@@ -1,77 +1,202 @@
...
 
-The most common issue when running our code is forgetting to source after
-opening a new terminal. For that, run the source commands in "Running the First
-Time" above, or use the shortcut
+In Python, the best way is to set up print() statements and print out relevant
+variables. Python is an interpreted language, meaning there's no easy way for
+us to step through it with a debugger.
 
-.. code-block:: bash
+In C++, you can achieve a similar print-by-debug with SPDLOG.
```
Comment (sid-parikh) [inline docs/source/faq.rst]: probably meant to say "debug-by-print"

File: docs/source/installation.rst [hunk 2]
```diff
@@ -1,13 +1,10 @@
...
 .. note:: 
    If you are completely unfamiliar with the command line or basic git
    usage, see the Tutorial page before proceeding. 
 
-Installation
-------------
-
 We only provide official support for Ubuntu 20.04 due to ROS2. Make sure you
 are on an Ubuntu 20.04 machine before continuing. Alternatively for
 Windows users, using WSL2 with Ubuntu 20.04 will work. The steps to set this
```
Comment (sid-parikh) [inline docs/source/installation.rst]: thoughts on adding a sentence about Macs? "For Mac users, Ubuntu 20.04 can be emulated in a virtual machine; on M1 macs using the arm64 version with the application UTM will work."

**Review by sid-parikh — CHANGES_REQUESTED** [review]
the one typo I commented about is worth fixing

**Review by sid-parikh — APPROVED** [review]

**Comment by Ninjatest321** [comment]
Would it be possible to maybe mention a resource where we can learn more about ROS and get a better understanding of it?

**Comment by kfu02** [comment]
> Would it be possible to maybe mention a resource where we can learn more about ROS and get a better understanding of it?

Generally, the tutorial links to all the relevant ROS docs. (There's not a great resource beyond that for learning how ROS works.)

EDIT: I'll put a sentence in the FAQ addressing that though.

File: docs/source/installation.rst (hunk 2 above)
Comment (kfu02) [reply docs/source/installation.rst]: sure, will add!

**Pull request merged**